storagenode/operator/diskspace.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package operator
```

Comment:
Maybe `dashboard` or `console`... calling it `operator` seems weird for some reason.